src/style.css

```diff
@@ -1,3 +1,125 @@
 body {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 400;
+  --button-color: rgba(0, 172, 220, 1);
+  --main-text-color: rgba(6, 11, 53, 1);
+  --secondary-text-color: rgba(97, 96, 112, 1);
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
+
+.card {
+  display: flex;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  width: 200px;
+
+  background-color: rgb(255, 255, 255);
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  align-items: center;
+  color: var(--main-text-color);
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+}
+
+.card__item-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-top: 40px;
+}
+
+.card__code {
+  margin-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--secondary-text-color);
+  align-self: flex-start;
+}
+
+.card__reviews {
+  margin-top: 16px;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.card__starbar {
+  display: flex;
+}
+
+.card__starbar--0 :nth-child(-n+0),
+.card__starbar--1 :nth-child(-n+1),
+.card__starbar--2 :nth-child(-n+2),
+.card__starbar--3 :nth-child(-n+3),
+.card__starbar--4 :nth-child(-n+4),
+.card__starbar--5 :nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.card__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.card__reviews-text {
+  font-size: 10px;
+  font-weight: 400;
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/ddb6b874-d3de-400a-97c4-0600c25a984e)
it means you can remove it